app/javascript/components/retirement-form/retirement-form.schema.js

```diff
@@ -139,6 +139,14 @@ const createSchema = (showTimeField, setShowTimeField) => ({
         or: [{ when: 'formMode', is: 'delay' }, { when: 'retirementDate', isNotEmpty: true }],
       },
     },
+    ...(showDateError ? [
+      {
+        id: 'dateWarning',
+        component: componentTypes.PLAIN_TEXT,
+        name: 'dateWarning',
+        label: __('Invalid date selected.'),
```

Comment:
@GilbertCherrie How about `Invalid date selected. Please select a future date.` ?

Reply:
Yeah I can change it